<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CENTAUR_NAMESPACE::LoginDialog</class>
 <widget class="QDialog" name="CENTAUR_NAMESPACE::LoginDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iniciar sesion</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:  rgb(3,12,19);
border: 0px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
}

QLabel{
font-size:  14px;
}
QLabel#serverLabel{
font-size:  25px;
}


QPushButton{
font-size: 14px;
font-weight: 600;
color:  #0D3C61;
border: 0px;
border-radius:  5px;
background-color: #2196f3;
}

QPushButton:default{
font-size: 14px;
font-weight: 800;
color:  #0d3C61;
border: 0px;
border-radius:  5px;
border: 2px solid rgba(144,203,249,100);
background-color: #2196f3;
}



QPushButton:hover, QPushButton:hover:default{
	background-color:  #339EF4;
}

QPushButton:pressed{
background-color:  #68B8F7;
}

QPushButton:pressed:default{
background-color:  #68B8F7;
}

QPushButton:disabled{
color: #c0c0c0;
background-color: #9D9EA4;
}

QLineEdit{
font-size: 14px;
border: 0px;
color: rgb(225,225,255);
background-color: rgb(63,70,76);
border-radius:  0px;
border-bottom: 2px solid  rgb(63,70,76);
min-height: 25px;
padding: 0 0 0 5;
}


QLineEdit:hover{
background-color:  rgb(78,85,90);
border-bottom: 2px solid #6ec6ff;
}

QLineEdit:focus{
color: white;
background-color:  rgb(94,100,105);
border-bottom: 2px solid rgb(33,150,243);
}

QGroupBox {
	font-size: 14px;
    background-color: rgb(26,34,40);
    border-radius: 5px;
    margin-top: 18px; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left; /* position at the top center */
    padding: 0 3px;
}


</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <widget class="QLabel" name="iconLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>156</width>
         <height>156</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1024</width>
         <height>768</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="CENTAUR_NAMESPACE::UserLineEdit" name="userEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
     <property name="placeholderText">
      <string>User</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CENTAUR_NAMESPACE::PasswordLineEdit" name="passwordEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>63</red>
           <green>70</green>
           <blue>76</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>63</red>
           <green>70</green>
           <blue>76</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>63</red>
           <green>70</green>
           <blue>76</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>63</red>
           <green>70</green>
           <blue>76</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>63</red>
           <green>70</green>
           <blue>76</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>63</red>
           <green>70</green>
           <blue>76</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>63</red>
           <green>70</green>
           <blue>76</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>63</red>
           <green>70</green>
           <blue>76</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>63</red>
           <green>70</green>
           <blue>76</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>225</red>
           <green>225</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="tfaEdit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>two-factor authentication</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="errLabel">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: #CF6679;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CENTAUR_NAMESPACE::UserLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../../Library/cui/include/UserLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>CENTAUR_NAMESPACE::PasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../../Library/cui/include/PasswordLineEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
